plugins/org.jkiss.dbeaver.ext.altibase.ui/META-INF/MANIFEST.MF

```diff
@@ -25,11 +25,13 @@ Require-Bundle: org.eclipse.core.runtime,
  org.jkiss.dbeaver.ext.altibase,
  org.jkiss.dbeaver.ui.editors.session,
  org.jkiss.dbeaver.tasks.native.ui,
- org.jkiss.dbeaver.tasks.ui
+ org.jkiss.dbeaver.tasks.ui,
+ org.jkiss.dbeaver.ext.ui.locks
 Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-17
 Bundle-Vendor: %Bundle-Vendor
 Bundle-ClassPath: .
 Bundle-Localization: OSGI-INF/l10n/bundle
 Export-Package: org.jkiss.dbeaver.ext.altibase.ui.views
 Automatic-Module-Name: org.jkiss.dbeaver.ext.altibase.ui
+Import-Package: org.jkiss.dbeaver.ext.ui.locks.edit

```

Comment:
You don't need to explicitly import a package. It's already available because you have a dependency on a bundle that exports it.

Reply:
> You don't need to explicitly import a package. It's already available because you have a dependency on a bundle that exports it.

Thanks for your comment! :) I can confirm that "org.jkiss.dbeaver.ext.ui.locks" is already included in the list of required bundles.

Have a great weekend.